cmd/exporter/main.go

```diff
@@ -185,9 +199,22 @@ func main() {
 	var issuer interface {
 		Issue(ctx context.Context) (string, error)
 	}
+	var standaloneIssuer *jwt.StandaloneTokenIssuer
+
 	switch {
 	case ctrlClient != nil:
 		issuer = jwt.NewToken(ctrlClient).For(slurmClusterID, "root").WithRegistry(jwt.NewTokenRegistry().Build())
+	case flags.standalone:
+		standaloneIssuer = jwt.NewStandaloneTokenIssuer(slurmClusterID, "root")
+		standaloneIssuer.WithScontrolPath(flags.scontrolPath)
```

Comment:
Why not use it since you made it possible?

```suggestion
		standaloneIssuer = jwt.NewStandaloneTokenIssuer(slurmClusterID, "root").WithScontrolPath(flags.scontrolPath)
```